docs/Serial.md

```diff
@@ -56,86 +56,179 @@ e.g. after configuring a port for GPS enable the GPS feature.
 
 You can use the CLI for configuration but the commands are reserved for developers and advanced users.
 
-The `serial` CLI command takes 6 arguments.
-
-1. Identifier (see serialPortIdentifier_e in the source)
-1. Function bitmask (see serialPortFunction_e in the source)
-1. MSP baud rate
-1. GPS baud rate
-1. Telemetry baud rate (auto baud allowed)
-1. Blackbox baud rate
-
-### Serial Port Identifier
-
- | Identifier | Value |
- | --- | ---: |
- | SERIAL_PORT_NONE | -1 |
- | SERIAL_PORT_USART1 | 0 |
- | SERIAL_PORT_USART2 | 1 |
- | SERIAL_PORT_USART3 | 2 |
- | SERIAL_PORT_UART4 | 3 |
- | SERIAL_PORT_UART5 | 4 |
- | SERIAL_PORT_USART6 | 5 |
- | SERIAL_PORT_USART7 | 6 |
- | SERIAL_PORT_USART8 | 7 |
- | SERIAL_PORT_LPUART1 | 8 |
- | SERIAL_PORT_USB_VCP | 20 |
- | SERIAL_PORT_SOFTSERIAL1 | 30 |
- | SERIAL_PORT_SOFTSERIAL2 | 31 |
- | SERIAL_PORT_IDENTIFIER_MAX | 31 |
-
-
-
-### Serial Port Function
-
- | Function | Value |
- | --- | ---: |
- | FUNCTION_NONE | 0 |
- | FUNCTION_MSP | 1 |
- | FUNCTION_GPS | 2 |
- | FUNCTION_TELEMETRY_FRSKY_HUB | 4 |
- | FUNCTION_TELEMETRY_HOTT | 8 |
- | FUNCTION_TELEMETRY_LTM | 16 |
- | FUNCTION_TELEMETRY_SMARTPORT | 32 |
- | FUNCTION_RX_SERIAL | 64 |
- | FUNCTION_BLACKBOX | 128 |
- | FUNCTION_TELEMETRY_MAVLINK | 512 |
- | FUNCTION_ESC_SENSOR | 1024 |
- | FUNCTION_VTX_SMARTAUDIO | 2048 |
- | FUNCTION_TELEMETRY_IBUS | 4096 |
- | FUNCTION_VTX_TRAMP | 8192 |
- | FUNCTION_RCDEVICE | 16384 |
- | FUNCTION_LIDAR_TF | 32768 |
- | FUNCTION_FRSKY_OSD | 65536 |
-
-
-
-### Baud Rates
-
-The allowable baud rates are as follows:
-
-| Identifier | Baud rate |
-| ---------- | --------- |
-| 0          | Auto      |
-| 1          | 9600      |
-| 2          | 19200     |
-| 3          | 38400     |
-| 4          | 57600     |
-| 5          | 115200    |
-| 6          | 230400    |
-| 7          | 250000    |
+The `serial` CLI command takes 6 arguments:
+```
+serial <port identifier> <port function> <msp baudrate> <sensor baudrate> <telemetry baudrate> <peripheral baudrate>
+```
 
+| Argument                       | Default | Source                   |
+| ------------------------------ | ------- | ------------------------ |
+| 1. Serial Port Identifier      |         | serialPortIdentifier_e   |
+| 2. Serial Port Function        |         | serialPortFunction_e     |
+| 3. Configuration/MSP baud rate | 115200  | baudrate_e               |
```

Comment:
What's the 'Source' column stand for?

Reply:
Like the original document it refers to variables in the source. I have removed it and put the original comments in the note. Also deleted the defaults column because I am not sure this code is setting this as default: https://github.com/betaflight/betaflight/blob/master/src/main/io/serial.c#L125-L136